lib/response.js

```diff
@@ -123,10 +123,10 @@ function newNote (req, res, next) {
   }
   models.Note.create({
     ownerId: owner,
-    alias: req.alias ? req.alias : null,
+    alias: req.alias ? req.alias : (config.allowFreeURL ? (req.params.alias ? req.params.alias : null) : null),
```

Comment:
This line is rather complex to read and also doesn't comply with the logic we use in another place:

https://github.com/codimd/server/blob/529075fd67ad0d41ba0b25a7de69817dd4cd76fb/lib/response.js#L161-L164

We should might do a proper abstraction of the code above and use it for both cases :)

Reply:
Sounds good, I'm working on it in the next few days.